ldelf/syscalls_rv.S

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023 ESWIN Corp. or its affiliates.
+ * Authors:
+ *   Liu Shiwei <liushiwei@eswincomputing.com>
+ */
+
+#include <asm.S>
+
+	.section .text
+
+	.macro LDELF_SYSCALL name, scn, num_args
+	FUNC \name , :
+
+	.if \num_args > 8
+	.error "Too many arguments for syscall"
+	.endif
+
+	li	t0, \scn
+	li	t1, \num_args
+	ecall
+	ret
+	END_FUNC \name
+	.endm
+
+	FUNC _ldelf_panic, :
```

Comment:
This shouldn't be indented, same below with `END_FUNC`
I know it is in for instance `ldelf/syscalls_a64.S`, but that's one of the few places and I guess it was by mistake.